docs/tutorial/deployments.md

```diff
@@ -146,29 +135,17 @@ if __name__ == "__main__":
 
 A few observations:
 
-- the `flow.to_deployment` interface exposes the *exact same* options as `flow.serve`; this method produces a deployment object
-- the deployments are only registered with the API once `serve(...)` is called
-- when serving multiple deployments, the only requirement is that they share a Python environment; they can be executed and scheduled independently of each other
-
-Spend some time experimenting with this setup.
-A few potential next steps for exploration include:
+- The `flow.to_deployment` interface exposes the *exact same* options as `flow.serve`; this method produces a deployment object.
+- Register deployments with the API after you call `serve(...)`.
+- When serving multiple deployments, the only requirement is that they share a Python environment; you can execute and schedule them independently of each other.
 
-- pausing and unpausing the schedule for the "sleeper" deployment
-- using the UI to submit ad-hoc runs for the "sleeper" deployment with different values for `sleep`
-- cancelling an active run for the "sleeper" deployment from the UI (good luck cancelling the "fast" one 😉)
+## Hybrid deployments
 
-!!! tip "Hybrid execution option"
-    Another implication of Prefect's deployment interface is that you can choose to use our hybrid execution model.
-    Whether you use Prefect Cloud or host a Prefect server instance yourself, you can run work flows in the environments best suited to their execution.
-    This model allows you efficient use of your infrastructure resources while maintaining the privacy of your code and data.
-    There is no ingress required.
-    For more information [read more about our hybrid model](https://www.prefect.io/security/overview/#hybrid-model).
+Prefect's deployments enable a hybrid execution model. This allows you to run workflows in Prefect Cloud or your own server, optimizing resource usage and data privacy. No inbound connections needed.
+For more information, see [hybrid model](https://www.prefect.io/security/overview/#hybrid-model).
 
 ## Next steps
 
 Congratulations! You now have your first working deployment.
 
-Deploying flows through the `serve` method is a fast way to start scheduling flows with Prefect.
-However, if your team has more complex infrastructure requirements or you'd like to have Prefect manage flow execution, you can deploy flows to a work pool.
-
-Learn about work pools and how Prefect Cloud can handle infrastructure configuration for you in the [next step of the tutorial](/tutorial/work-pools/).
+However, if your team has more complex infrastructure requirements or you'd like to have Prefect manage flow execution, you can [deploy flows to a work pool](/tutorial/work-pools/).
```

Comment:
I think it's worth an explicit ask to continue to the next step of the tutorial here.